components/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -156,6 +156,18 @@ class D2LFilterDropdown extends mixinBehaviors([D2L.PolymerBehaviors.FilterDropd
 		}
 	}
 
+	setFilterOptions(categoryKey, options) {
```

Comment:
This function is a convenience function. I could have used `removeFilterOption` to remove every key but this seemed like it deserved it's own helper

Reply:
Makes sense!  As discussed offline, I want to eventually remove these public methods and just break up the components in a way that they will take the data directly, and if the HM filter's data changes the UI will automatically update, instead of having to be told to update its own copy of the data

Reply:
I agree I would say the ideal is it taking data, and would update itself automatically

Reply:
We could definitely do it as an "in-between", but I'd rather just do the full refactor that avoids passing a huge data object into a component